apps/src/templates/projects/ProjectWidget.jsx

```diff
@@ -5,18 +5,45 @@ import ContentContainer from '../ContentContainer.jsx';
 import i18n from "@cdo/locale";
 import _ from 'lodash';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import Button from "../Button";
+import color from "../../util/color";
 
-const ProjectWidget = React.createClass({
-  propTypes: {
+const styles = {
+  button: {
+    float: 'right',
+    marginTop: 10,
+  },
+  headingStartNew: {
+    paddingRight: 10,
+    paddingBottom: 10,
+    fontSize: 16,
+    fontFamily: '"Gotham 4r"',
+    color: color.charcoal,
+    width: 940
```

Comment:
Is 940 on purpose? Can we use 970: https://github.com/code-dot-org/code-dot-org/blob/staging/shared/css/style-constants.scss#L14

Reply:
this style was useless (possibly because I've added `clear: 'both'` in places) so I've removed it.